### PR TITLE
Feature/bc 643 hide old task overview

### PR DESCRIPTION
**Pull request by Uzaeir**
# Description
Removing the old task overview

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-643
https://github.com/hpi-schul-cloud/nuxt-client/pull/1992
https://github.com/hpi-schul-cloud/schulcloud-client/pull/2733
https://github.com/hpi-schul-cloud/dof_app_deploy/pull/71

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.hpi-schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: features/dashboard/task.feature [hunk 1]
```diff
@@ -17,8 +17,10 @@ Feature: dashboard test set
 		And <userRole> clicks on Private-task checkbox
 		And <userRole> clicks Add-task-submit button
 		And <userRole> goes to tasks page
-		And <userRole> filter by '<courseName>'
-		When <userRole> sees that task with name '<taskName>' is visible on the list
+		And <userRole> clicks on drafts tab
+		#And <userRole> filter by '<courseName>'
+		Then <userRole> sees '<taskName>' in the list
+		#When <userRole> sees that task with name '<taskName>' is visible on the list
```
Comment (d3n153) [inline features/dashboard/task.feature:23]: We should create a ticket to remove the step "#Then <userRole> should see that task with name '<taskName>' is not visible on the list" and also the methods and so on behind it.
If we aren't able to solve the filter issue, it should be added as well.

File: page-objects/pages/TASKListPage.js [hunk 2]
```diff
@@ -154,19 +153,47 @@ async function clickDeleteTaskButtonInPopup() {
 	await driver.refresh();
 }
 
-async function clickAtTask(){
+async function clickAtTask(taskName) {
```
Comment (d3n153) [inline page-objects/pages/TASKListPage.js:156]: Do we need both clickOnTask(button) | clickAtTask(taskName) or can we maybe combine them? Can be done as follow up.

**Review by odalys-dataport — APPROVED** [review]

File: page-objects/pages/TASKListPage.js [hunk 3]
```diff
@@ -154,19 +153,47 @@ async function clickDeleteTaskButtonInPopup() {
...
+	const taskOverviewResult = await getNuxtTaskList();
+	const taskIndex = taskOverviewResult.indexOf(taskName);
+	let clickOnTask = taskOverviewResult[taskIndex];
+	const taskInTheList = '//div[text() =' + "'" + clickOnTask + "'" + ']';
+	return taskInTheList;
 }
 
-async function taskInCompletedTabDisplayed(){
-	await elementHelpers.isElementDisplayed(taskAtCompletedTab);
+//#taskDisplayed and taskNotDisplayed doesn't seem to work
```
Comment (d3n153) [inline page-objects/pages/TASKListPage.js:172]: Would need to check this as well. Also possible as follow up.

File: page-objects/pages/TASKPage.js [hunk 4]
```diff
@@ -167,13 +163,16 @@ async function clickCompletedTab(){
 	await elementHelpers.clickAndWait(completedTaskTab);
 }
 
+async function clickDraftsTab(){
```
Comment (d3n153) [inline page-objects/pages/TASKPage.js:166]: We should add this method as well:
async function clickOpenTasksTab(){
	await elementHelpers.clickAndWait(openTaskTab);
}

const openTaskTab= "//span[@data-testid = 'openTasks']";

File: page-objects/pages/TASKListPage.js (hunk 2 above)
Comment (d3n153) [reply page-objects/pages/TASKListPage.js:156]: Will be done within BC-1008.

File: page-objects/pages/TASKListPage.js (hunk 3 above)
Comment (d3n153) [reply page-objects/pages/TASKListPage.js:172]: Will be done within BC-1008.

File: page-objects/pages/TASKPage.js (hunk 4 above)
Comment (d3n153) [reply page-objects/pages/TASKPage.js:166]: Will be done within BC-1008.

File: features/dashboard/task.feature (hunk 1 above)
Comment (d3n153) [reply features/dashboard/task.feature:23]: Will be done within BC-1008.

**Pull request merged**